_posts/2017-12-30-new-years-rust-a-call-for-community-blogposts.md

```diff
@@ -0,0 +1,70 @@
+# New Year's Rust: A Call for Community Blogposts
+
+Tis the season for people and communities to reflect and set goals- and the Rust team is
+no different. Last week, we published [a blogpost about our accomplishments in 2017](rust-2017),
+and the teams have already begun brainstorming goals for next year. One thing that we know for
+certain is that Rust is commited to extending and improving our community organization and
+outreach- and this post is just the first of what we hope to be many iterations of new kinds of
+community feedback mechanisms.
+
+[rust-2017]: https://blog.rust-lang.org/2017/12/21/rust-in-2017.html
+
+Last year, the Rust team started a new tradition: defining a roadmap of goals for the 
+upcoming year. We leveraged our 
+[RFC process](https://github.com/aturon/rfcs/blob/roadmap-2017/text/0000-roadmap-2017.md)
+to solicit community feedback. While we got a lot of awesome feedback on that RFC, we'd 
+like to try something new in addition to the RFC process: a call for community blog posts
+for ideas of what the goals should be.
```

Comment:
Can you add a sentence or two about why we want blog posts, instead of just comments on a 2018 RFC roadmap? Maybe something along the lines of "we have heard from some people can find the formal RFC process intimidating, since it can be difficult to form concrete plans-of-actions from looser user experiences. We want to hear from everyone from language theorists to people struggling to learn the language."